pages/returning-members.js

```diff
@@ -0,0 +1,282 @@
+import React, { useState, useContext, useEffect, useRef } from 'react';
+import { isEmail } from "validator";
+import { ThemeContext } from "styled-components";
+import PageTemplate from '../frontend/components/templates/PageTemplate';
+import { SystemComponent, SystemSpan } from '../frontend/components/atoms/SystemComponents';
+import Subtitle from "../frontend/components/atoms/Subtitle";
+import Card from '../frontend/components/atoms/Card';
+import FieldSection from "../frontend/components/molecules/Form/FieldSection";
+import Button from '../frontend/components/atoms/Button';
+
+const TERMDESCRIPTIONS = [
+"1A Co-op", 
+"1B Study", 
+"1B Co-op", 
+"2A Study", 
+"2A Co-op", 
+"2B Study", 
+'2B Co-op',
+"3A Study", 
+"3A Co-op",
+"3B Study", 
+"3B Co-op", 
+"4A Study",
+"4A Co-op", 
+"4B Study", 
+"4B Co-op", 
+"Graduate Student", 
+"Not currently enrolled at the University of Waterloo", 
+"Alumni",
+];
+
+const PREVTERMS = ['F22', 'S22', 'W22','F21', 'S21', 'W21', 'F20', "S20",'W20', 'F19', 'S19', 'W19', 'F18', 'S18', 'W18'];
```

Comment:
For this constant and the ones below it, insert an underscore between the different words `PREV_TERMS`.